app/views/users/_form2.html.erb

```diff
@@ -1,5 +1,5 @@
 <div style="padding:10px;">
-<%= form_for :user, :as => :user, :url => :register + "?return_to=" + request.fullpath, :html => {:class => "row"} do |f| %>
+<%= form_for :user, :as => :user, :url => register_url + "?return_to=" + request.fullpath, :html => {:class => "row"} do |f| %>
```

Comment:
@oorjitchowdhary verify that this line is same. I have recently pushed the changes.

Reply:
I've to change this.. let me see again then